rules/windows/discovery_adfind_command_activity.toml

```diff
@@ -0,0 +1,64 @@
+[metadata]
+creation_date = "2020/10/19"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/10/19"
+
+[rule]
+author = ["Elastic"]
+description = """
+This rule detects the Active Directory query tool, AdFind.exe. AdFind has legitimate purposes, but it is frequently
+leveraged by threat actors to perform post-exploitation Active Directory reconnaissance. The AdFind tool has been
+observed in Trickbot, Ryuk, Maze, and FIN6 campaigns. This rule requires Sysmon.
```

Comment:
```suggestion
observed in Trickbot, Ryuk, Maze, and FIN6 campaigns.
```
Hey @peasead, what is the Sysmon wording referring to? I believe this rule should work out of the box with Elastic endpoint and winlogbeat

Reply:
Thanks Dan.

This doesn't fire for Winlogbeat without Sysmon. I can clarify that Sysmon is only needed for Winlogbeat and not Endpoint.
With Sysmon:
![image](https://user-images.githubusercontent.com/7442091/101504225-94109a80-3938-11eb-8559-049fc91ec9bc.png)
Without Sysmon:
![image](https://user-images.githubusercontent.com/7442091/101505266-c8388b00-3939-11eb-9b72-e8135e441124.png)
Endpoint:
![image](https://user-images.githubusercontent.com/7442091/101505313-d686a700-3939-11eb-96dd-cf0e68c2ec20.png)